TASVideos.Common/ITimeable.cs

```diff
@@ -42,6 +42,11 @@ public static string ToStringWithOptionalDaysAndHours(this TimeSpan timeSpan)
 			return timeSpan.ToString(@"h\:mm\:ss\.ff");
 		}
 
+		if (timeSpan.TotalSeconds == 0.0)
+		{
+			return "00:00.01";
+		}
+
 		return timeSpan.ToString(@"mm\:ss\.ff");
```

Comment:
Instead of `if (timeSpan.TotalSeconds == 0.0)` this should be `if (timeSpan.TotalSeconds <= 0.01)`, beacuse otherwise a movie time of 0.002s would still be shown as 00:00.00.